src/serac/numerics/mesh_utils.cpp

```diff
@@ -167,6 +167,96 @@ std::shared_ptr<mfem::ParMesh> buildCuboidMesh(int elements_in_x, int elements_i
   return std::make_shared<mfem::ParMesh>(comm, mesh);
 }
 
+std::shared_ptr<mfem::ParMesh> buildCylinderMesh(int radial_refinement, int elements_lengthwise, double radius,
+                                                 double height, const MPI_Comm comm)
+{
+  static constexpr int dim                   = 2;
+  static constexpr int num_vertices          = 17;
+  static constexpr int num_elems             = 12;
+  static constexpr int num_boundary_elements = 8;
+
+  // a == 1.0 produces a mesh of a cylindrical "core" surrounded by
+  // a cylindrical "shell", but makes some of the elements in the "core" section
+  // nearly degenerate
+  //
+  // a > 1 makes the "core" section no longer a cylinder, but its elements
+  // are no long nearly degenerate
+  constexpr double        a                           = 1.3;
+  static constexpr double vertices[num_vertices][dim] = {{0.0000000000000000, 0.0000000000000000},
```

Comment:
Out of curiosity, how did you generate these disk vertices?

Reply:
Here's the mathematica script that generated them:
![cross_section](https://user-images.githubusercontent.com/61714427/106032853-daa38b00-6085-11eb-8d87-2f82f1af7c9c.png)

The first "ring" radius was chosen to be (sqrt(3) / 3) so that each element in the cross section has the same area.

Reply:
@joshessman-llnl No, I think this is separate. I mostly just added the cuboid-related mesh options to the lua mesh input schema.

Reply:
@samuelpmishLLNL In the future, would it be difficult to make a "hollow" cylinder (e.g. delete the core)? Maybe just specify with an internal radius? That might also be useful.

Reply:
Sure, I'll add another one later today: `buildHollowCylinder` that takes an inner and outer radius and similar other parameters.